constructorio-client/src/test/java/io/constructor/client/ConstructorIOFacetOptionConfigurationTest.java

```diff
@@ -177,12 +177,12 @@ public void testCreateFacetOptionConfigurations() throws Exception {
         assertEquals("test-option", jsonOption1.get("value"));
         assertEquals("test-alias", jsonOption1.get("value_alias"));
         assertEquals("Test Option", jsonOption1.get("display_name"));
-        assertEquals(1, jsonOption1.get("position"));
+        assertEquals(Integer.valueOf(1), jsonOption1.get("position"));
```

Comment:
Can we add a similar test to this where we set values to null, see if it gets set correctly?